test/test-start.js

```diff
@@ -0,0 +1,37 @@
+var assert = require("assert"); // assert module is supported from nodejs.

```

Comment:
Please add a boilerplate at head of this file as follows:
// Copyright (c) 2017 The Absolute Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.